db_holder.py

```diff
@@ -0,0 +1,41 @@
+from peewee import *
+
+db = SqliteDatabase('facts.db')
+
+
+class Fact(Model):
+    factName = CharField()
```

Comment:
we use snake_case for variables and function names - fact_name, return_data, etc.